types/eslint/index.d.ts

```diff
@@ -597,14 +597,22 @@ export namespace Rule {
         report(descriptor: ReportDescriptor): void;
     }
 
+    type ReportFixer = ((fixer: RuleFixer) => null | Fix | IterableIterator<Fix> | Fix[]);
```

Comment:
Is the wrapping `()` needed here?

Reply:
it isn't, i had copied it from the previous code. will update, good catch 👍